README.md

```diff
@@ -1,2 +1,53 @@
-# tic-tac-toe-oop-ruby
-A Tic Tac Toe game written in Ruby
+<!-- PROJECT LOGO -->
+# Tic-Tac-Toe-game
+A Ruby implementation of the loved Tic Tac Toe game.
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+- [Tic-Tac-Toe-game](#tic-tac-toe-game)
+  - [Table of Contents](#table-of-contents)
+- [About The Project](#about-the-project)
+  - [Tic Tac Toe in Ruby](#tic-tac-toe-in-ruby)
+  - [How to play](#how-to-play)
+  - [Features:](#features)
+    - [Built on Ruby](#built-on-ruby)
+    - [Interactive prompt system](#interactive-prompt-system)
+  - [Built With](#built-with)
+- [Getting started](#getting-started)
+  - [Installation](#installation)
+- [Contact](#contact)
+- [Acknowledgements](#acknowledgements)
+<!-- ABOUT THE PROJECT -->
+# About The Project
+## Tic Tac Toe in Ruby
+This is a Tic Tac Toe game, developed following the OOP paradigm and built in Ruby.
+## How to play
+* Read all instructions the game prompts carefully
+* Player 1 chooses the symbol they want to play with, player 2 will get assigned the remaining symbol
+* A prompt appears asking for the position that player 1 wants to move to
+* Once player 1 selects, player 2 will be prompted to select a position
+* The game alternates between both players until the board is full or until someone wins
```

Comment:
What are the conditions for winning?
Consider using images to show winning combinations.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone4/readme.png)